src/shogun/lib/SGVector.h

```diff
@@ -389,23 +383,6 @@ template<class T> class SGVector : public SGReferencedData
 		 */
 		void add(const T x);
 
-		/** Addition operator */
-		SGVector<T> operator+ (SGVector<T> x);
-
-		/** Inplace addition operator */
-		SGVector<T> operator+= (SGVector<T> x)
-		{
-			add(x);
-			return *this;
-		}
-
-		/** Inplace addition operator for sparse vector */
-		SGVector<T> operator+= (SGSparseVector<T>& x)
```

Comment:
btw is this defined in linalg?

Reply:
good point. There is no specialisation for SGSparseVector in linalg..

Reply:
Should I let this remain as it was then? And only replace the SGVector calls?

Reply:
I think so

Reply:
@gf712 If no compiler errors happen, I would just remove as unused?

Reply:
I guess so. Just thinking if there is something hiding behind a `#ifdef`..

Reply:
I say remove and fix the errors when they appear :) We want to clean up these classes in any case ...

Reply:
OK, in that case can merge once the CI is fixed, just to be sure

Reply:
one thing I am always concerned about is changing code that is not covered by any tests (at least integration) .... but in this case I think we can justify it?